src/graphics/canvas.rs

```diff
@@ -207,25 +207,15 @@ impl Drawable for Canvas {
 
         // We have to mess with the scale to make everything
         // be its-unit-size-in-pixels.
-        let scale_x = param.src.w * f32::from(self.width());
-        let scale_y = param.src.h * f32::from(self.height());
+        let scale_x = param.src.w * self.width() as f32;
+        let scale_y = param.src.h * self.height() as f32;
```

Comment:
Why use ```as``` here instead of ```from```?

Reply:
habit, always used as instead of from, should probably be changed to be consistent with the rest of the library